### PR TITLE
Allow using CALL together with PeriodicCommit

### PR DESCRIPTION
**Pull request by pontusmelke**

### CODE REVIEW

**Comment by jakewins** [comment]
:+1: Happy to merge if @magnusvejlstrup is +1

File: integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java [hunk 1]
```diff
@@ -523,6 +538,77 @@ public void shouldGracefullyFailWhenSpawningThreadsCreatingTransactionInProcedur
...
+        Result result = db.execute( "USING PERIODIC COMMIT 1 " +
+                                    "LOAD CSV FROM '" + url + "' AS line " +
+                                    "CALL org.neo4j.procedure.createNode(line[0]) YIELD node as n " +
+                                    "RETURN n.prop" );
+        // THEN
+        for (int i = 1; i <= 100; i++)
+        {
+            assertThat( result.next().get( "n.prop" ), equalTo( Integer.toString( i ) ) );
+        }
+    }
```
Comment (davidegrohmann) [inline integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java:566]: I think that we should also check the state of the database after the write query and that we have committed 100 times.

Comment (pontusmelke) [inline integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java:566]: yes we should

**Comment by pontusmelke** [comment]
@davidegrohmann added a check

**Comment by boggle** [comment]
@pontusmelke ok by me, looks good

**Pull request merged**